eng/versioning/version_client.txt

```diff
@@ -32,7 +32,7 @@ com.azure:azure-cosmos-encryption;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-data-appconfiguration;1.1.5;1.1.6
 com.azure:azure-data-schemaregistry;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-data-schemaregistry-avro;1.0.0-beta.4;1.0.0-beta.5
-com.azure:azure-data-tables;12.0.0-beta.1;12.0.0-beta.2
+com.azure:azure-data-tables;12.0.0-beta.2;12.0.0-beta.2
```

Comment:
If this is a `prepare release PR` we shouldn't be changing the dependency version.